entries/t/travelers.com.json

```
@@ -0,0 +1,17 @@
+{
+  "travelers.com": {
```

The official name seems to be Travelers?
```suggestion
  "Travelers": {
```